testing/test_cacheREADME.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import, division, print_function
+import _pytest
+
+pytest_plugins = ("pytester",)
+
+
+class Helper(object):
+
+    def check_readme(self, testdir):
+        config = testdir.parseconfigure()
+        readme = config.cache._cachedir.join("README.md")
+        return readme.isfile()
+
+
+class TestReadme(Helper):
+
+    def test_readme_passed(self, testdir):
+        testdir.tmpdir.join("test_a.py").write(
```

Comment:
It's easier to use `testdir.makepyfile`:

```python
testdir.makepyfile("""
    def test_always_passes():
        assert 1
""")
```

Reply:
got it